### DISCUSSION TITLE
Osu!mobile touch box edit

### DISCUSSION THREAD

**Discussion by MnMgt5234** [ppy/osu#28586] [Ideas] [answered]
![Screenshot_20240625-153256](https://github.com/ppy/osu/assets/173802084/6d43b91d-c557-439c-b39a-c991e3ec4378)
I want the touch boxes to be arranged on mobile, such as opacity and size.

**Comment by Stoppedpuma** [comment, marked as answer]
Already planned, see https://github.com/ppy/osu/discussions/28361#discussioncomment-9757867

**Reply by MnMgt5234** [reply]
![Screenshot_20240715-122054](https://github.com/user-attachments/assets/5433daef-b46f-4b95-a8bb-1f08fa1fe865)
So will there be any optimizations for Android?

**Reply by Tenexxt** [reply]
It's probably going to be worked on later, like WAY later. The team has a lot planed for Lazer, so improvements for optimisation on a platform most don't know you can play on isn't a priority

**Reply by MnMgt5234** [reply]
I see
